Add copy about ordering exercises in exercises.md (#129)
exercism/docs: language-tracks/configuration/exercises.md

This lets people optimize the learning experience to their own interests—skipping topics that they are not interested in, or that they already know a lot about, and doing deeper dives into topics that they're curious about.
Take a look at the (non-exhaustive) [topics list][topics] for suggestions of topics to add.

## Order of exercises

> Exercises that are not part of the core track (`"core": false`) should be added to `config.json` on a line between the last core exercise and the end of the file (i.e. all core exercises are located at the top of the file, above all side exercises).

Reviewer: ```suggestion
Exercises that are not part of the core track (`"core": false`) should be added to `config.json` below the last core exercise and the end of the file (i.e. all core exercises are located at the top of the file, above all side exercises).
```

Reviewer 2: Perhaps still a little bit confusing, how can it be both below the last core exercise and below the end of the file?  Should it be below the last core at the end of the file?

Reviewer 3: Wouldn't it be enough to say after the last core exercise.  Do we really need to say before the end of the the file?

Reviewer 2: Right. I can put things (technically speaking) after the end of a file (post EOF character), but that doesn't mean I should.  Perhaps leave out mention of the end of the file altogether?

Reviewer: Ha. My edit wasn't processed correctly. There's not meant to be a between anymore. This is the real one:

```suggestion
Exercises that are not part of the core track (`"core": false`) should be added to `config.json` below the last core exercise (i.e. all core exercises are located at the top of the file, above all side exercises).
```
